caikit/interfaces/ts/data_model/util.py

```diff
@@ -0,0 +1,83 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Third Party
+import pandas as pd
+
+# Local
+import caikit.interfaces.ts.data_model as dm
+
+
+# pylint: disable=too-many-return-statements
+def mts_equals(left: dm.TimeSeries, right: dm.TimeSeries, **kwargs) -> bool:
```

Comment:
Is it worth defining `__eq__` directly on `dm.TimeSeries`?

Reply:
Done.